templates/zerver/help/organization-logo-as-favicon.md

```diff
@@ -0,0 +1,13 @@
+# Use organization logo as browser favicon
+
+This is feature has not yet been implemented.
+
+### Move the users list
```

Comment:
This was probably just a leftover header from copy/paste, but I'd change to 'Set browser favicon' or something along those lines.